cfg/DB/func/get_log.ddl

```diff
@@ -26,7 +26,44 @@ select logs.id as logId,
                                   ' в компанию ', companies.name)
 
                   ELSE '' END,
-              CASE WHEN log_actions.is_insert_text = true THEN CONCAT(' ', logs.text) ELSE '' END
+              CASE WHEN log_actions.is_insert_text = true THEN
+                       CASE WHEN (log_actions.name = 'CODE_LOG_UPDATE'
+                           AND logs.text LIKE '{%}'
+                           AND logs.text::json is not null)
+                                THEN
+
+                                (SELECT DISTINCT array_to_string(array_agg(CONCAT(
```

Comment:
You don't used POstreSQL json function?

Reply:
I use some, it is a select from json_each()
But I do not use json_to_record, it don't meet my needs